workflows/VGP-assembly-v2/Plot-Nx-Size/Generate-Nx-and-Size-plots-for-multiple-assemblies.ga

```diff
@@ -0,0 +1,689 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
```

Comment:
I think you want to say Person in the editor interface here, then you can also put your orcid id